src/components/AddressMetadataForm.tsx

```diff
@@ -51,7 +51,7 @@ const AddressMetadataForm = ({
         <>
           <HorizontalDivider narrow />
           <InlineLabelValueInput
-            label={t`★ Main address`}
+            label={`★ ${t`Main address`}`}
```

Comment:
Since we already had the `Main address` keyword, I removed the `★ Main address` one.